connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -45,6 +52,9 @@
     private static String sPaymentCancelUri;
     private static String sPaymentSuccessUri;
     private static SdkProfile sdkProfile;
+    private static ConnectivityManager connectivityManager;
+    private static Network cellularNetwork;
```

Comment:
Just a control question, since these are initalised only once... are you sure they do not need any reinitialisation if the user changes something on the device (turns off WiFi, ejects SIM card, inserts another SIM or similar)?